README.md

```diff
@@ -36,6 +36,8 @@ export VAGRANT_DEFAULT_PROVIDER=libvirt
 vagrant up
 ```
 
+**Important:** The Vagrantfile create an internal network so you must not run Vagrant in parallel because the internal network can be created multiple times leading to undefined behavior. With the Libvirt provider, VMs are launch in parallel so, if you want speed, use Libvirt provider.
```

Comment:
Did you have issues with https://github.com/TOSIT-IO/tdp-getting-started/blob/master/scripts/vagrant-up-parallel.sh?

Reply:
Yes, when I run it I saw `vboxnet0` and `vboxnet1` with the same cidr for the network.

Reply:
Vagrant can create networks, write file like Ansible inventory and if you run it in parallel you will have undefined behavior. Vagrant is not designed to be run like that. I think we should advise the use of Libvirt instead Virtualbox.